### ISSUE TITLE
Support for 'backdropFilter' CSS property

### ISSUE THREAD

**Issue by chrisdrackett** [necolas/react-native-web#1070]
I understand limiting styles to those that react-native supports, but sometimes you want to use an unsupported style in web to get functionality you can get in react-native but isn't available in web.

For example I want to use `WebkitBackdropFilter` to simulate vibrancy on web. However when I try and use this style I get an error. Not sure the solution here, open to suggestions!

**Comment by rickysullivan** [comment]
https://facebook.github.io/react-native/docs/platform-specific-code

**Comment by chrisdrackett** [comment]
right, but I'm using react-native-web and my target is safari, so I can use `WebkitBackdropFilter` without issue other than I get a error from react-native-web. I can see a reason to give this error on react-native, but react-native-web is different. For example, react-native does not have the idea of `cursor` but I'm able to use it in react-native-web without warnings

**Comment by rickysullivan** [comment]
Couldn't you just do something like this:

```
const styles = StyleSheet.create({
  logoImageFilter:
  Platform.OS === "web"
    ? {
        filter: "saturate(150%)"
      }
    : {}
  })
```

**Comment by chrisdrackett** [comment]
sure, this is what i'm doing, but RNW is still throwing errors in my console:

```Warning: Failed prop type: Invalid props.style key `WebkitBackdropFilter` supplied to `View`.```

**Comment by flip-it** [comment]
I would also want `backdropFilter` to not throw error. 

Inline styles are actually also validated, @giuseppeg.

**Comment by necolas** [comment]
I'll consider adding it when this issue is resolved https://github.com/rofrischmann/inline-style-prefixer/issues/164. But the [feature request for Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=497522) has been open since 2015, so I don't think this is going to be widely available any time soon

**Comment by necolas** [comment]
This will be included in 0.10.0
